### PR TITLE
Monday code corrections

### PR DESCRIPTION
**Pull request by DocJek**
fixed problem with DOM tree but now can't get lists to populate

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,37 +1,34 @@
...
+  getCustPerHour:  function() {
+    this.custPerHour = randNum(this.minCust, this.maxCust);
   },
   getCookiesPerHour: function() {
-    return Math.floor(this.getCustPerHour() * this.avgCookieSales);
+    firstAndPike.getCustPerHour();
+    this.cookiesPerHour = Math.floor(this.custPerHour * this.avgCookieSales);
+    for (var i = 0; i < 16; i++) {
+      var cookie = firstAndPike.getCookiesPerHour();
+      custAndCookiesPerHour.push(cookie[i]);
```
Comment (traceyr) [inline js/app.js:22]: you have a linter error here that may help you. When we refer to another key/pair value inside an object, what do we need to put in front of it(hint: this).

Comment (traceyr) [reply js/app.js:22]: You actually have 34 linter issues and I think they may help you fix your code!

File: js/app.js [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
+}
 
 var firstAndPike = {
   name: 'First and Pike',
   minCust: 23,
   maxCust: 65,
   avgCookieSales: 6.3,
-  getCustPerHour: function() {
-    return randNum(this.minCust, this.maxCust);
+  custAndCookiesPerHour: [],
```
Comment (traceyr) [inline js/app.js:13]: This is great!

File: js/app.js [hunk 3]
```diff
@@ -45,7 +42,7 @@ function buildArray1() {
   var ul = document.createElement('ul');
   article.appendChild(ul);
 
-  for (var x = 0; x < custAndCookiesPerHour.length; x++) {
+  for (var x = 0; x < firstAndPike.custAndCookiesPerHour.length; x++) {
     var li = document.createElement('li');
     li.textContent = custAndCookiesPerHour[x];
```
Comment (traceyr) [inline js/app.js:47]: custAndCookiesPerHour is part of the object. What do we need to do to access a property inside of the object?

**Comment by DocJek** [comment]
code wasn't working, fixed on different push

**Pull request closed without merging**